src/ObjectSchema.test.js

```diff
@@ -804,6 +804,31 @@ describe('ObjectSchema', () => {
     })
   })
 
+  describe('only', () => {
+    it('returns a subset of the object', () => {
+      const base = S.object()
```

Comment:
Not sure it's really needed but can you add a nested object too. Just to check we don't miss anything.

Reply:
Can you add in the README an example of this use case? 
... and thank you for the nice addition! 🙏